awx/conf/settings.py

```diff
@@ -86,49 +87,46 @@ def _ctit_db_wrapper(trans_safe=False):
             if is_atomic:
                 rollback_set = transaction.get_rollback()
                 if rollback_set:
-                    logger.debug('Obtaining database settings in spite of broken transaction.')
```

Comment:
@AlanCoding you don't want to recurse these, because this can just result in _more_ DB lookup issues.

Reply:
We had a blacklist of loggers for the filter class used on the external logger handler. It's possible that the blacklisting logic might be needed for this new stuff as well. There are a good handful of places it applies to.